workloads/vars/logging.yml

```diff
@@ -18,4 +18,10 @@ ORIGIN_TESTS_VERSION: "{{ lookup('env', 'ORIGIN_TESTS_VERSION')|default('latest'
 CLUSTER_LOADER_STRING: "{{ '[sig-scalability][Feature:Performance][Serial][Slow] Load cluster'
                             if ORIGIN_TESTS_VERSION == 'latest'
                             or ORIGIN_TESTS_VERSION is version('4.4', '>=')
-                            else '[Feature:Performance][Serial][Slow] Load cluster' }}"
\ No newline at end of file
+                            else '[Feature:Performance][Serial][Slow] Load cluster' }}"
+
+FLUENT_CONF: "{{ lookup('env', 'FLUENT_CONF')|default('files/logging-fluentd-wo-infra.conf', true) }}"
+
+KUBECONFIG: "{{ lookup('env', 'KUBECONFIG')|default(lookup('env','HOME') + '/.kube/config') }}"
+# Running with ansible_connection=local may alter this location
+WORKLOAD_DIR: "{{ lookup('env', 'WORKLOAD_DIR')|default(lookup('env','PWD')) }}"
```

Comment:
Unless I am mistaken, the default here won't work on a remote host because var files are evaluated on the host machine

Reply:
is this any different then before?  For example, the KUBECONFIG line is basically the same which was in the playbook.  Doesn't ansible ssh to the host and run those tasks remotely on the control host meaning say the KUBECONFIG file would need to be the same as well?

I'm open for suggestions.  I really would like to be able to run from my laptop at the location of my choosing so this value should accomplish that and allow the original still to be satisfied, though I think the original should default to HOME and not PWD

Reply:
You're right about KUBECONFIG being the same and moving that to the var file is a nice change.

I think changing WORKLOAD_DIR default to $HOME fixes both of our issues